src/ai/ai.controller.ts

```
@@ -0,0 +1,80 @@
+    };
+  }
+
+  /**
+   * Make the AI play a move for this game and persist it.
+   * Returns the updated game state.
+   */
+  @Post(':id/ai-move')
+  async aiMove(
+    @Param('id') gameId: string,
+    @Body() body: AiMoveBodyDto,
+  ) {
```

ESLint

[error] Replace `⏎····@Param('id')·gameId:·string,⏎····@Body()·body:·AiMoveBodyDto,⏎··` with `@Param('id')·gameId:·string,·@Body()·body:·AiMoveBodyDto`

(prettier/prettier)